WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BadgeView.swift

```diff
@@ -98,13 +106,18 @@ private extension BadgeView {
         case .circle:
             Circle()
                 .fill(customizations.backgroundColor)
-                .stroke(Color.white, lineWidth: Layout.borderLineWidth)
+                .overlay(
+                    Circle()
+                        .stroke(Color.white, lineWidth: Layout.borderLineWidth)
+                        .opacity(customizations.bordered ? 1 : 0)
+                )
```

Comment:
Nit: why don't we apply this only if `bordered` is true?


```suggestion
                .if(customizations.bordered) { view in
                    view.overlay(
                        Circle()
                            .stroke(Color.white, lineWidth: Layout.borderLineWidth)
                    )
                }
```

Similar suggestion for the case below.

Reply:
Done [here](https://github.com/woocommerce/woocommerce-ios/pull/16307/commits/620f93742d0741b99bb2dbde05500270147ce488)